File: official/guides/customs-guide/csharp/buy-international-shipment.cs
shipment.Buy(shipment.LowestRate(
  includeServices: new List<string>() { "First" },
  excludeCarriers: new List<string>() { "USPS" }
));

// OR

shipment.Buy("{RATE_ID}");

// Print PNG link
System.Console.WriteLine(shipment.postage_label.label_url);
// Print Tracking Code
System.Console.WriteLine(shipment.postage_label.tracking_code);

File: official/guides/customs-guide/ruby/buy-international-shipment.rb
shipment.buy(
  rate: shipment.lowest_rate(['USPS'], ['First']),
)

# #OR

shipment.buy(rate: { id: '{RATE_ID}' })

## Print PNG link
puts shipment.postage_label.label_url

## Print Tracking Code
puts shipment.postage_label.tracking_code
